modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -117,6 +117,10 @@ class RadialReturnStressUpdate : public StressUpdateBase, public SingleVariableR
   MaterialProperty<Real> & _effective_inelastic_strain;
   const MaterialProperty<Real> & _effective_inelastic_strain_old;
   Real _max_inelastic_increment;
+
+  const RankTwoTensor _itwo;
```

Comment:
Add doxygen descriptions of these: (and maybe name them like `_ident_two` to make it a little more clear from their names what they are)
```
/// Rank two identity tensor
const RankTwoTensor _itwo;
```